examples/node-app-mssql/package.json

```diff
@@ -5,7 +5,8 @@
     "lint": "eslint src/index.js",
     "lint:fix": "eslint --fix src/index.js",
     "sync-db": "sync-db",
-    "start": "yarn lint && node src/index.js"
+    "start": "yarn lint && node src/index.js",
+    "docker-start": "yarn sync-db && yarn start"
```

Comment:
No need this here. I think you can put this directly in the `Dockerfile` so that the users know the commands explicitly.

Reply:
Done